### PR TITLE
chore: add dbmate to the contrib guide

### PR DESCRIPTION
**Pull request by deniseli**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ brew install orbstack
 ```
 or [OrbStack Website](https://orbstack.dev/)
 
+We use [dbmate](https://github.com/amacneil/dbmate) to manage migrations, so please install that if
+you don't have it already:
+```
+brew install dbmate
+```
```
Comment (wesbillman) [inline CONTRIBUTING.md:24]: Legend!! Thank @deniseli 

Should we add a example usage somewhere as well? I def ran into this at some point and just kind of guessed :)

**Review by wesbillman — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ brew install orbstack
 ```
 or [OrbStack Website](https://orbstack.dev/)
 
+We use [dbmate](https://github.com/amacneil/dbmate) to manage migrations, so please install that if
+you don't have it already:
+```
+brew install dbmate
```
Comment (alecthomas) [inline CONTRIBUTING.md:23]: This is included in Hermit, so remove this bit.

**Review by alecthomas — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (deniseli) [reply CONTRIBUTING.md:24]: Great idea! Done!

File: CONTRIBUTING.md (hunk 2 above)
Comment (deniseli) [reply CONTRIBUTING.md:23]: Ahh nice! Removed

**Pull request merged**